packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1444,26 +1444,24 @@ class _SliverOffstageElement extends SingleChildRenderObjectElement {
 /// Mark a child as needing to stay alive even when it's in a lazy list that
 /// would otherwise remove it.
 ///
-/// This widget is for use in a [RenderAbstractViewport]s, such as
-/// [Viewport] or [TwoDimensionalViewport].
-///
-/// This widget is rarely used directly. The [SliverChildBuilderDelegate] and
-/// [SliverChildListDelegate] delegates, used with [SliverList] and
-/// [SliverGrid], as well as the scroll view counterparts [ListView] and
-/// [GridView], have an `addAutomaticKeepAlives` feature, which is enabled by
-/// default, and which causes [AutomaticKeepAlive] widgets to be inserted around
-/// each child, causing [KeepAlive] widgets to be automatically added and
-/// configured in response to [KeepAliveNotification]s.
-///
-/// The same `addAutomaticKeepAlives` feature is supported by the
+/// This widget is used in [RenderAbstractViewport]s, such as [Viewport] or
+/// [TwoDimensionalViewport], to manage the lifecycle of widgets that need to
+/// remain alive even when scrolled out of view.
+///
+/// The [SliverChildBuilderDelegate] and [SliverChildListDelegate] delegates,
```

Comment:
The fact that some widgets in the framework already have this built-in is really valuable I think. Can we mention it above in the other areas where AutomaticKeepAlive comes up?

Reply:
I added it in [Add mention of addAutomaticKeepAlives](https://github.com/flutter/flutter/pull/168311/commits/b2409541828aab06907ed55c39dc4cda22feb2b6). It felt like having a template for it would be overkill, so I copied and pasted. Tell me what you think about it